tests/core/test_expectation_suite.py

```diff
@@ -237,6 +237,7 @@ def profiler_config():
     return yaml.load(yaml_config)
 
 
+@pytest.mark.unit
 def test_expectation_suite_equality(baseline_suite, identical_suite, equivalent_suite):
     """Equality should depend on all defined properties of a configuration object, but not on whether the *instances*
     are the same."""
```

Comment:
`assert baseline_suite is baseline_suite` can be removed

Reply:
this comment applies throughout

Reply:
I feel we are abusing test fixtures here, and it's reducing the readability of the test